dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/Router.java

```diff
@@ -51,4 +51,31 @@
      */
     <T> List<Invoker<T>> route(List<Invoker<T>> invokers, URL url, Invocation invocation) throws RpcException;
 
+    /**
+     * priority
+     *
+     * @return
+     */
+    Integer getPriority();

```

Comment:
I think `int` is better, let's rely on auto-boxing.

Reply:
Ok, modified, thank you